Dangerfile

```diff
@@ -0,0 +1,10 @@
+# Sometimes its a README fix, or something like that - which isn't relevant for
+# including in a CHANGELOG for example
+has_code_changes = !git.modified_files.grep(/\.go$/).empty?
+
+# Let people say that this isn't worth a CHANGELOG entry in the PR if they choose
+declared_trivial = (github.pr_title + github.pr_body).include?("#trivial") || !has_code_changes
```

Comment:
This logic here seems nice and simple.
Please add a document to (README??) about `#trivial`.

Actually, I don't think it is best to dump everything to the README but don't know where els eto place dev-level docs